install/kubernetes/helm/istio/charts/prometheus/templates/_affinity.tpl

```diff
@@ -0,0 +1,93 @@
+{{/* affinity - https://kubernetes.io/docs/concepts/configuration/assign-pod-node/ */}}
```

Comment:
This commit deserves a detailed commit message about why we're adding `_affinity.tpl` temporarily to this fork, but not adding it to the PR upstream.

Reply:
done